server/neptune/workflows/internal/pr/workflow.go

```diff
@@ -1,10 +1,42 @@
 package pr
 
-import "go.temporal.io/sdk/workflow"
+import (
+	internalContext "github.com/runatlantis/atlantis/server/neptune/context"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/activities"
+	tfModel "github.com/runatlantis/atlantis/server/neptune/workflows/activities/terraform"
+	workflowMetrics "github.com/runatlantis/atlantis/server/neptune/workflows/internal/metrics"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/notifier"
+	"go.temporal.io/sdk/workflow"
+	"strconv"
+	"time"
+)
 
 const TaskQueue = "pr"
 
-func Workflow(ctx workflow.Context, request Request) error {
-	// TODO: implement workflow
-	return nil
+type prActivities struct {
+	*activities.Github
+}
+
+func Workflow(ctx workflow.Context, request Request, tfWorkflow TFWorkflow) error {
+	var a *prActivities
+	options := workflow.ActivityOptions{
+		TaskQueue:           TaskQueue,
+		StartToCloseTimeout: 5 * time.Second,
+	}
+	ctx = workflow.WithActivityOptions(ctx, options)
+	ctx = workflow.WithValue(ctx, internalContext.RepositoryKey, request.RepoFullName)
+	ctx = workflow.WithValue(ctx, internalContext.PullNumKey, request.PRNum)
```

Comment:
This should be included already as per: https://github.com/lyft/atlantis/blob/865248e6ad2616a742de7d828945aa09b60863d6/server/neptune/temporal/propagator.go#L18-L19